### ISSUE TITLE
Sencha Touch Android - Notification is never created

### ISSUE THREAD

**Issue by ds8k** [katzer/cordova-plugin-local-notifications#381]
After the most recent push to master I was able to build an Android apk and hit the "add" function without having the app crash. The problem now is that the notification is never actually created. I can add a notification that should immediately pop up (like in the README), however it never happens. If I add a notification with a future date and then check getScheduled it says no notifications are scheduled. Here's where I call the add function:

```
window.plugin.notification.local.add(
                {
                    id: '1',
                    message: 'You left something behind!',
                    date: date
                },
                function() {
                    console.log('Notification added');
                }
            );
```

This shows up in the logcat:

```
E/LocalNotification-Add(26374): The given arguments does not contain a JSONArray
D/>>>>>>>>><<<<<<<<<<<>>>>>>>>>><<<<<<<<Action(26374): trackAction
I/chromium(26374): [INFO:CONSOLE(1)] "Notification added", source: file:///android_asset/www/app.js (1)
```

Clearly this is how the most recent push fixed the crashing, however it didn't fix the real problem. For what it's worth, the plugin works perfectly on iOS.

**Comment by PKnittel** [comment]
Should be fixed now.

**Comment by ds8k** [comment]
Notifications are now triggering but they're ignoring the scheduled date. Also tapping on the notification doesn't appear to clear it from the window shade.

**Comment by ds8k** [comment]
Found the commit that broke the date:

https://github.com/katzer/cordova-plugin-local-notifications/commit/e60410899631a04da794e31c5a2d80cba74c45c7#diff-ad04e476620fb3e555a517cea124bbd1R488

Rolling back the mergeWithDefaults function to how it was before fixed scheduling on both Android and iOS.

**Comment by katzer** [comment]
@ds8k, had [fixed](a15cb52972a8299cc733e0a10e92fa145197e254) the bug some minutes later after your mentioned commit.

**Comment by katzer** [comment]
The new 0.8.x release is a complete rewrite. If the issue still exists please reopen it.

You may also take a look into the new [wiki](https://github.com/katzer/cordova-plugin-local-notifications/wiki).

Thanks!
